### PR TITLE
Single output file

### PR DESCRIPTION
**Pull request by alegosto01**
benchmarks fix and bcftools' rule implementation

### CODE REVIEW

File: workflow/Snakefile [hunk 1]
```diff
@@ -121,20 +139,55 @@ rule samtools_index:
...
+	resources: 
+			mem_mb= 32000,
+			runtime= 1440   
+	shell:
+        	"cuteSV {input.bam} {input.gen} {output} results/"
+rule bcftools:
+	input:
+			sniffles="results/sniffles/patient_sniffles.vcf",
+			cutesv="results/cutesv/patient_cutesv.vcf",
+			bash="resources/variants_analysis.sh"
```
Comment (tdido) [inline workflow/Snakefile]: Scripts should not be declared as inputs. If the script is simple enough it should be added directly into the `shell` directive. If it's more complex (which I suspect is the case) then you should replace `shell` with the `script` directive. Docs here: https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#bash

**Comment by tdido** [comment]
You're now missing the script in the repo, you need to add it and commit.

**Comment by alegosto01** [comment]
now the script is in the resources folder that which is in gitignore, do i need to save it in a different location?

**Comment by tdido** [comment]
Yes, in `workflow/scripts`. See https://snakemake.readthedocs.io/en/stable/snakefiles/deployment.html for details.

File: workflow/scripts/variants_analysis.sh
Comment (tdido) [inline workflow/scripts/variants_analysis.sh:1]: Input and output paths should be dynamic, not hardcoded. Check the example in the [Snakemake docs](https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#bash-example-1)

**Review by tdido — APPROVED** [review]

**Pull request merged**